.github/workflows/ci.yaml

```diff
@@ -127,14 +129,14 @@ jobs:
         cp dist/ca.pem "${scratch}/ca.pem"
 
         echo "##[group]Build"
-          cat hack/boilerplate.ytt.txt > "${scratch}/config/cartogrpaher-conventions.yaml"
-          ko resolve -B -f dist/cartogrpaher-conventions.yaml >> "${scratch}/config/cartogrpaher-conventions.yaml"
-          kbld -f "${scratch}/config/cartogrpaher-conventions.yaml" --imgpkg-lock-output "${scratch}/.imgpkg/images.yml" > /dev/null
+          cat hack/boilerplate.ytt.txt > "${scratch}/config/cartographer-conventions.yaml"
+          ko resolve -B -f dist/cartographer-conventions.yaml >> "${scratch}/config/cartographer-conventions.yaml"
```

Comment:
not reaaaally related to the PR, but I noticed I had to bump here to go 1.18 - do you think we should bump carto's `go.mod`'s go version to enforce 1.18 (and use 1.18 in ci too), or hold off for a bit? _(i imagine the answer to be "yep, bump it")_

Reply:
now actually related: for the samples, we're doing `KO_DOCKER_REPO="${KO_DOCKER_REPO}/samples"` while here straight `ko resolve`

<img width="691" alt="Screen Shot 2022-03-22 at 4 52 31 PM" src="https://user-images.githubusercontent.com/3574444/159573851-05d82e03-2887-4f4f-a869-59d308cea43b.png">
 
is there a specific reason for having to set `KO_DOCKER_REPO=${...}` in there?

Reply:
> do you think we should bump carto's go.mod's go version to enforce 1.18

Wait for 1.18.1 to play it safe. I bumped for this project since we use multiple modules and workspaces are available.

Reply:
> is there a specific reason for having to set `KO_DOCKER_REPO=${...}` in there?

We're publishing images to a local registry and then exporting to a bundle tar file, so the actual repo the image is published to doesn't matter. It will all end up in the tar file and eventual relocated registry just the same. I'll open a separate PR to drop that bit since it's noise.

Reply:
done